consultation_analyser/support_console/export_url_guidance.py

```diff
@@ -0,0 +1,67 @@
+import logging
+
+import boto3
+import pandas as pd
+from django.conf import settings
+
+from consultation_analyser.consultations.models import Consultation, QuestionPart, Respondent
+
+logger = logging.getLogger("export")
+
+
+def get_response_id_mapping(s3_key: str) -> dict:
+    # Get the response ID mapping
+    s3 = boto3.client("s3")
+    response = s3.get_object(Bucket=settings.AWS_BUCKET_NAME, Key=s3_key)
+    answer_df = pd.read_excel(response["Body"].read(), sheet_name="Sheet2")
```

Comment:
I take it this bit gets the themefinder_id right? (The outputs are correct!)

Just seems a bit magical that e.g. ID 7 is row 9.